controllers/networking-calico/charts/internal/calico/values.yaml

```diff
@@ -6,7 +6,10 @@ config:
     type: "host-local"
 #    subnet: "usePodCidr"
 images:
-  calico-node: "quay.io/calico/node:v3.7.2"
-  calico-cni: "quay.io/calico/cni:v3.7.2"
-  calico-typha: "quay.io/calico/typha:v3.7.2"
-  calico-kube-controllers: "quay.io/calico/kube-controllers:v3.7.2"
+  calico-node: "quay.io/calico/node:v3.8.2"
```

Comment:
Please don't use exact image names + tags here. The images should always come from the image vector.
Use `calico-node: image:tag` (etc.) instead.

Reply:
the images do come from the ImageVector, I ma just following the practice here. We could of course replace these values with any other but what would you recommend to have here?

Reply:
Nothing, a place holder, sth like https://github.com/gardener/gardener-extensions/blob/master/controllers/provider-aws/charts/internal/machine-controller-manager/seed/values.yaml#L2.
It's confusing to double-maintain the values here.